frontend/src/components/tickets-form/index.tsx

```diff
@@ -9,36 +9,58 @@ import { SelectedProductsWithVariationsList } from "./selected-products-with-var
 import { Ticket } from "./types";
 
 type Props = {
+  isBusiness: boolean;
   tickets: Ticket[];
   selectedProducts: SelectedProducts;
   addProduct: (id: string, variant?: string) => void;
   removeProduct: (id: string, variant?: string) => void;
 };
 
 export const TicketsForm: React.SFC<Props> = ({
+  isBusiness,
   tickets,
   selectedProducts,
   addProduct,
   removeProduct,
-}) => (
-  <React.Fragment>
-    {tickets.map(ticket => (
-      <Box key={ticket.id}>
-        <ProductRow
-          quantity={
-            selectedProducts[ticket.id] ? selectedProducts[ticket.id].length : 0
-          }
-          ticket={ticket}
-          addProduct={addProduct}
-          removeProduct={removeProduct}
-        />
-      </Box>
-    ))}
-
-    <SelectedProductsWithVariationsList
-      selectedProducts={selectedProducts}
-      products={tickets}
-      removeProduct={removeProduct}
-    />
-  </React.Fragment>
-);
+}) => {
+  const ticketsToShow = tickets.filter(ticket => {
+    if (ticket.variations!.length > 0) {
+      return true;
+    }
+
+    if (isBusiness && ticket.name.toLowerCase().includes("business")) {
+      return true;
+    }
+
+    if (!isBusiness && !ticket.name.toLowerCase().includes("business")) {
+      return true;
+    }
+
+    return false;
+  });
+
+  return (
+    <React.Fragment>
+      {ticketsToShow.map(ticket => (
+        <Box key={ticket.id}>
+          <ProductRow
+            quantity={
+              selectedProducts[ticket.id]
+                ? selectedProducts[ticket.id].length
+                : 0
```

Comment:
this is something I wrote do you think we can do `selectedProducts?.length ?? 0`?